cdap-ui/app/cdap/components/Experiments/ExperimentsServiceControl/index.js

```diff
@@ -45,12 +46,13 @@ export default class ExperimentsServiceControl extends Component {
       shouldStopService: false,
       artifactName: MMDSArtifact,
       api: myExperimentsApi,
-      i18nPrefix: ''
+      i18nPrefix: PREFIX
     }).subscribe(
       this.props.onServiceStart,
-      () => {
+      (err) => {
         this.setState({
-          error: T.translate(`${PREFIX}.errorMessage`),
+          error: err.error,
+          extendedError: err.extendedMessage,
```

Comment:
This is not guaranteed to be a string. Rendering this in DOM will result in a JS error (react cannot render objects in DOM)